presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -95,6 +95,15 @@ session properties are included.
 
 .. _tuning-memory:
 
+``per-query-retry-limit``
+^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Minimum value:** ``0``
+* **Default value:** ``0``
+
+The number of times that a query is automatically retried in the case of a transient query or communications failure. 
```

Comment:
Let's also mention that the default of 0 indicates that it means retry is disabled (might be obvious, but better to make it obvious).